mara_db/format.py

```diff
@@ -0,0 +1,53 @@
+"""Different formats for piping"""
+
+
+class Format:
+    """Base format definition"""
+
+    def __repr__(self) -> str:
+        return (f'<{self.__class__.__name__}: '
+                + ', '.join([f'{var}={getattr(self, var)}'
+                             for var in vars(self) if getattr(self, var)])
+                + '>')
+
+
+class CsvFormat(Format):
```

Comment:
Maybe also add a `Native` e.g. in case of pg -> pg pipelines?

Reply:
Done.